ruby_event_store/spec/mappers/default_spec.rb

```diff
@@ -0,0 +1,46 @@
+SomethingHappened = Class.new(RubyEventStore::Event)
+
+module RubyEventStore
+  module Mappers
+    RSpec.describe Default do
+      let(:data)   {{some_attribute: 5}}
+      let(:metadata)     {{some_meta: 1}}
+      let(:domain_event) { SomethingHappened.new(data: data, metadata: metadata) }
+
+      specify '#event_to_serialized_record returns YAML serialized record' do
+        record = subject.event_to_serialized_record(domain_event)
+        expect(record).to            be_a SerializedRecord
+        expect(record.id).to         eq domain_event.event_id
```

Comment:
please verify that it equals that manually provided UUID that I asked for above :)

Reply:
👍